ducktape/src/main/scala/io/github/arainko/ducktape/internal/modules/MaterializedConfiguration.scala

```diff
@@ -90,6 +90,12 @@ private[ducktape] object MaterializedConfiguration {
         val sourceFieldName = Selectors.fieldName(Fields.source, sourceSelector)
         Product.Renamed(destFieldName, sourceFieldName) :: Nil
 
+      case '{ FieldConfig.default[source, dest, destFieldType]($destSelector)(using $ev1, $ev2) } =>
+        val destFieldName = Selectors.fieldName(Fields.dest, destSelector)
+        val field = Fields.dest.byName.get(destFieldName).getOrElse(Failure.abort(Failure.InvalidFieldSelector(destSelector, summon, Suggestion.fromFields(Fields.dest))))
+        val default = field.default.getOrElse(Failure.abort(Failure.DefaultMissing(field.name, Type.of[dest]))) 
```

Comment:
as I mentioned under the test, you might want to check if the default can be assigned to that field with `default.asTerm.tpe <:< TypeRepr.of(using field.tpe)` and provide a new subclass of `Failure` just for that